test/integration/targets/cloudscale_volume/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/cloudscale
+shippable/cloud/group1
```

Comment:
just missed, there should be an `unsupported` in the alias file, and the ' shippable/cloud/group1' could be removed

Reply:
I changed this now.